domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casetype/CaseTypeEntityACLValidatorImplTest.java

```diff
@@ -1,20 +1,23 @@
 package uk.gov.hmcts.ccd.definition.store.domain.validation.casetype;
 
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnit;
-import org.mockito.junit.MockitoRule;
+
 import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationResult;
 import uk.gov.hmcts.ccd.definition.store.repository.entity.AccessProfileEntity;
 import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeACLEntity;
 import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeEntity;
 
+import org.junit.Rule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
```

Comment:
I think that the `org.junit.Rule`, `MockitoJunit` and `MockitoRule` imports are for initialising mocking with JUnit4. I believe these can be removed and replaced with the `@ExtendWith(MockitoExtension.class)`. This would also mean that `openMocks()` in `setup()` isn't needed.